src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/ILaunchProfileExtensionValueProviderMetadata.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Microsoft.VisualStudio.ProjectSystem.Debug
+{
+    /// <summary>
+    /// Metadata mapping interface for the <see cref="ExportLaunchProfileExtensionValueProviderAttribute"/>.
+    /// </summary>
+    public interface ILaunchProfileExtensionValueProviderMetadata
+    {
+#pragma warning disable CA1819 // Properties should not return arrays
```

Comment:
Why not use IReadOnlyCollection instead of suppressing this?

Reply:
This is consistent with the other places we define types for collecting export attribute metadata. That's not really a good _technical_ reason, of course. :-)

I'll look into `IReadOnlyCollection` and see if that would be possible.

Reply:
My understanding is that these need to be arrays in order to match the properties in the related attributes.